cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterLifecycleTests.java

```diff
@@ -217,28 +217,6 @@ public void testInvalidConfigReturns400() throws Exception {
     // TODO: implement once adapter creation calls configureTemplate()
   }
 
-  @Test
```

Comment:
This test is being removed since we don't deploy the template again if the JAR contents have not changed and thus this test is not correct.